README.md

```diff
@@ -77,15 +77,21 @@ This installation guide is for ISIS users interested in installing ISIS (3.6.0)+
 
         conda config --env --add channels usgs-astrogeology
 
+        # If you have followed conda-forges advice to set `channel_priority=strict`,
+        # you need to set it to flexible for the install:
+        conda config --set channel_priority flexible
+
 
 6.  The environment is now ready to download ISIS and its dependencies:
 
-        conda install -c usgs-astrogeology isis
+        # To install e.g. version 4.3
+        conda install isis=4.3
 
     If you would like to work with our latest ISIS version 3, rather than updating
     to ISIS 4, instead run:
 
-	    conda install -c usgs-astrogeology isis=3.10.0
+	    # This will find the latest 3.x.y release
```

Comment:
This is redundant.

Reply:
Don't understand. My version has the advantage of staying correct even when you'd release 3.10.1 and more?

Reply:
We're going to release any more 3.x versions. 3.10.2 was the final ISIS 3 release.

Reply:
Sorry, my comment wasn't about the new command, it is 👍 .

I do not like having text that says something and then immediately after it having an in-line comment that says the same thing. So, I think you should remove the comment line.